app/assets/javascripts/search.coffee

```diff
@@ -43,19 +43,20 @@ $(document).ready ->
     switch recordType
       when 'variables'
         $searchButtonSpan.text('Search Variables').addClass('variable-text')
+      when 'services'
+        $searchButtonSpan.text('Search Services').addClass('service-text')
       else
-        $searchButtonSpan.text('Search Collections').removeClass('variable-text')
+        $searchButtonSpan.text('Search Collections')
 
 
   $('#record_type_collections').on 'click', ->
-    $("#search-submit-button-text").text('Search Collections').removeClass('variable-text')
+    $("#search-submit-button-text").text('Search Collections').removeClass('variable-text').removeClass('service-text')
```

Comment:
going with the above comment, seems inconsistent that for collections you are removing two classes, but for variables and services you are adding a class and removing a class